tests/Manipulator/ObjectManipulatorTest.php

```diff
@@ -88,6 +88,14 @@ public function testSetObject(): void
         ObjectManipulator::setObject($instance, $object, $fieldDescription);
     }
 
+    public function testSetObjectWithoutMappedBy(): void
+    {
+        $fieldDescription = $this->createMock(FieldDescriptionInterface::class);
+        $fieldDescription->expects($this->once())->method('getAssociationMapping')->willReturn(['mappedBy' => null]);
+
+        ObjectManipulator::setObject(new \stdClass(), new \stdClass(), $fieldDescription);
```

Comment:
Tests above perform expectations about the methods present in the object passed as argument 1 to `ObjectManipulator::setObject()`. Shouldn't we do the same here?

Reply:
Here the method is not called. (And doesn't exist)

Reply:
Fine. I was thinking about something like
```php
$instance->expects($this->never())->method('theMethod')->with($object);
```
but since I haven't enough context, I trust your decision if you say it is not required.